graphql/multiplication-connect/updateUserGamesPlayed.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from "@apollo/client";
+
+export const UPDATE_USER_GAMES_PLAYED_MCDATA = gql`
+  mutation updateUserData($id: String = "wHZGkjB7CFf0cMrcGgI1pIHfke02") {
+    update_multiplicationConnectData(
+      where: { id: { _eq: $id } }
```

Comment:
maybe you could use the userId from the users table here as well

Reply:
Do you have an example of this in one of your files?